src/PhylloConnect.ts

```diff
@@ -14,6 +14,7 @@ interface IPhylloInitialize {
   userId: string
   environment: PhylloEnvironment
   workPlatformId?: string
+  singleAccount?: Boolean
```

Comment:
Why is it exposed?

Reply:
Akshay, this is in the React native package code, not in the example folder.